cosmos/dbt_adapters/bigquery.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from typing import Any
+
+from cosmos.exceptions import CosmosValueError
+
+
+def _mock_bigquery_adapter() -> None:
+    from typing import Optional, Tuple
+
+    import agate
+    from dbt.adapters.bigquery.connections import BigQueryAdapterResponse, BigQueryConnectionManager
+    from dbt_common.clients.agate_helper import empty_table
+
+    def execute(  # type: ignore[no-untyped-def]
+        self, sql, auto_begin=False, fetch=None, limit: Optional[int] = None
```

Comment:
```suggestion
        self, sql, auto_begin=False, fetch=None, limit: int | None = None
```

Reply:
I copied the signature from the dbt_adapter as is defined there since we wish to mock that function. I think it's have the signature as is there to avoid any type mismatch errors. WDYT?